### DISCUSSION TITLE
CLI serverless runtime has incorrect "reserved" local environment variables (AWS)

### DISCUSSION THREAD

**Discussion by wceriale** [vercel/community#3173] [Bug Report]
### Page to Investigate

https://vercel.com/docs/concepts/projects/environment-variables/reserved-environment-variables#allowed-environment-variables

### Steps to Reproduce

To recreate:
- Use `vercel dev` with any of the following [environment variables](https://vercel.com/docs/concepts/projects/environment-variables/reserved-environment-variables#allowed-environment-variables) set in `env.local`.

Temporary fix:
- To avoid throwing the error, remove the allowed variables from: \
https://github.com/vercel/fun/blob/main/src/index.ts#L43
- To copy environment variables to the serverless runtime, remove the allowed variables from: \
https://github.com/vercel/fun/blob/main/src/providers/native/index.ts#L97

**Comment by smaeda-ks** [comment]
@wceriale could you create a minimal reproducible example repo?
https://vercel.com/guides/creating-a-minimal-reproducible-example
